lib/logger/lib/logger/utils.ex

```diff
@@ -153,7 +153,7 @@ defmodule Logger.Utils do
   ## encoding
 
   defp collect_cc(:encoding, [?l | t], args, used_format, used_args, opts),
-    do: collect_cc(:done, t, args, [?l | used_format], used_args, %{opts | char_lists: false})
+    do: collect_cc(:done, t, args, [?l | used_format], used_args, %{opts | charlists: false})
```

Comment:
I don't understand what this is trying to do, what function does this opts get passed to? If it's inspect, then `false` is not a valid option, maybe it's `:as_lists` is what needs to be used..